models/issue_comment.go

```diff
@@ -976,6 +980,12 @@ func (opts *FindCommentsOptions) toConds() builder.Cond {
 	if opts.Type != CommentTypeUnknown {
 		cond = cond.And(builder.Eq{"comment.type": opts.Type})
 	}
+	if opts.Line > 0 {
```

Comment:
Are there database index on this columns? If not, this will be slow when the database is big.

Reply:
You only need one index per a query. ReviewID has an index.